lighthouse-cli/test/smokehouse/test-definitions/pwa/pwa-expectations.js

```diff
@@ -102,7 +102,7 @@ module.exports = [
           score: 1,
         },
         'works-offline': {
-          score: 0,
+          score: 1,
```

Comment:
This does work offline. Was this meant to be testing something subtle re: redirection? I'm assuming not, since no comments.

Reply:
I'm guessing it didn't *always* work offline, this test might be 4 years old :)